pkg/controller/cluster/cluster_controller.go

```diff
@@ -1,22 +1,33 @@
 package cluster
 
 import (
+	// "context"
```

Comment:
Why is this commented out at the top, and then present two lines below?
[NOTE: This is also testing that I can interact with Github]